test/Microsoft.SqlTools.ServiceLayer.UnitTests/Profiler/ProfilerServiceTests.cs

```diff
@@ -65,5 +65,62 @@ public class ProfilerServiceTests
             Assert.Equal(sessionListener.PreviousSessionId, sessionId);
             Assert.Equal(sessionListener.PreviousEvents.Count, 1);
         }
+
+        /// <summary>
+        /// Test stopping a session and receiving event callback
+        /// </summary>
+        /// <returns></returns>
+        [Fact]
+        public async Task TestStopProfilingRequest()
```

Comment:
It looks like the code coverage number dropped a bit with this PR.  It could be unrelated flakyness, but please verify it isn't related to a test failure from this change.